WooCommerce/Classes/ViewRelated/Products/Edit Product/DefaultProductFormTableViewModel.swift

```diff
@@ -386,20 +382,13 @@ private extension DefaultProductFormTableViewModel {
                 .map({ String.localizedStringWithFormat(format, $0.name, $0.options.count) })
                 .joined(separator: "\n")
         default:
-            if isAddProductVariationsEnabled {
-                details = ""
-            }
-            else {
-                details = Localization.variationsPlaceholder
-            }
+            details = ""
```

Comment:
is this change intentional? if so, maybe `Localization.variationsPlaceholder` can be removed?